docs/docs/cmd/spo/file/file-checkin.md

```diff
@@ -14,7 +14,7 @@ m365 spo file checkin [options]
 : The URL of the site where the file is located
 
 `-f, --url [url]`
-: The server-relative URL of the file to retrieve. Specify either `url` or `id` but not both
+: The server-relative or site-relative URL of the file to retrieve. Specify either `url` or `id` but not both
```

Comment:
We can make this more readable by changing to `The server- or site-relative URL...`. Let's change all similar occurrences in other files.